packages/amplication-data-service-generator-runner/Dockerfile

```diff
@@ -1,6 +1,19 @@
-FROM node:19.1-alpine AS base
+FROM node:18.13.0-slim AS base
```

Comment:
`node:18.13.0-slim` is debian based and it won't have `apk` package manager. 
We either need to use the alpine version or fix line 8. 

My suggestion is to revert to an alpine based image and update the user configuration https://wiki.alpinelinux.org/wiki/Setting_up_a_new_user